Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -25,29 +25,40 @@ typedef NS_ENUM(NSInteger, RCTPositionErrorCode) {
   RCTPositionErrorTimeout,
 };
 
+typedef NS_ENUM(NSInteger, RCTLocationPermissionCode) {
+  RCTLocationPermissionIOSUnknown = 1,
```

Comment:
since we are creating this number based enum, i'd like us to create a corresponding "enum" object in javascript so people don't have to know these magic numbers. Alternatively you can make the option a string that represents the enum, and parse it on the native side. Either way, i would want the end user to not have to know any magic numbers